packages/overdrive/lib/components/Tabs/TabList.tsx

```diff
@@ -118,6 +119,8 @@ export const TabList: FunctionComponent<Props> = ({
 
 	return (
 		<Box
```

Comment:
The indication bubble thingo isn't optically aligned to the middle of its label.

![image](https://user-images.githubusercontent.com/599459/88354363-f9a62500-cda3-11ea-9d05-e9475abb4685.png)

https://www.chromatic.com/snapshot?appId=5c80456c1498920020dedf4e&id=5f1a3d85f4366400229d2623

Reply:
Done